.github/workflows/cd.yml

```diff
@@ -89,6 +89,10 @@ jobs:
 
           result-encoding: string
 
+      - name: Support multi-platform
+        run: |
+          sed -i 's/FROM /FROM --platform=$BUILDPLATFORM /g' ./Dockerfile
```

Comment:
Could we use `buildargs` for these?

Reply:
in this case, actually we don't need to pass `buildargs` as its already handled.

> instead of writing a constant platform value into our Dockerfile we should use a variable instead, FROM --platform=$BUILDPLATFORM debian
https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/